### ISSUE TITLE
Contact Actions: Skype Lite actions are missing

### ISSUE THREAD

**Issue by pipip** [BranchMetrics/sesame_issues#191]
Hello,
I bought the Contact Action Addon some time ago and really like it so far.
Recently, I started using Skype again and decided to use the  Skype Lite application.

The contact Actions plugin lists two actions but only to send a SMS (text via skype) and to make a regular phone call (dialer via skype lite). All the skype actions are missing. The regular contact app is listing them tho (Message, Voice Call and Video Call). 
Could you please add the Skype lite actions to the plugin?

**Comment by philwall2** [comment]
Yep. We'll take a look. Fair warning though: if Sesame isn't detecting them now, it may be that Skype Lite doesn't support shortcuts to those actions.

Doesn't hurt to investigate though. Thanks for the feedback!
